activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -93,6 +93,17 @@ def column_exists?(table_name, column_name, type = nil, options = {})
                                       (!options.key?(:null)      || c.null == options[:null]) }
       end
 
+      # Returns just a table's primary key
+      def primary_key(table_name)
+        pks = primary_keys(table_name)
+        return nil unless pks.size == 1
+        pks.first
```

Comment:
(Also, prefer Ruby expressions and conditionals over early-return control flow, e.g. `pks.first if pks.one?`